Makefile

```diff
@@ -1,6 +1,45 @@
+.PHONY: clean upgrade test quality
+# Generates a help message. Borrowed from https://github.com/pydanny/cookiecutter-djangopackage.
+help: ## display this help message
+	@echo "Please use \`make <target>\` where <target> is one of"
+	@perl -nle'print $& if m{^[\.a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## delete generated byte code and coverage reports
+	find . -name '*.pyc' -delete
+	find . -name '__pycache__' -type d -exec rm -rf {} ';' || true
+	rm -rf coverage htmlcov
+	rm -rf assets
+	rm -rf pii_report
+
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -q -r requirements/pip_tools.txt
 	pip-compile --upgrade -o requirements/pip_tools.txt requirements/pip_tools.in
 	pip-compile --upgrade -o requirements/base.txt requirements/base.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
+	pip-compile --upgrade -o requirements/test.txt requirements/test.in
+
+requirements: ## install development environment requirements
+	pip install -q -r requirements/pip_tools.txt
+	pip install -qr requirements/base.txt --exists-action w
+	pip-sync requirements/base.txt requirements/constraints.txt requirements/test.txt requirements/travis.txt
+
+quality-js: ## Run JavaScript linter
+	echo "Tool not configured for quality target"
```

Comment:
If it's not setup for JS Quality, just remove the target so we can find repos with this issue in the future.